src/pipe.toit

```diff
@@ -348,7 +348,7 @@ Can be passed either a command (with no arguments) as a
 Throws an exception if the command cannot be run or if the command exits
   with a signal, but otherwise returns the exit value of the program.
 */
-run_program arguments:
+run_program arguments -> int?:
```

Comment:
This actually never returns null, because exit_code will only return null if there was a signal, but in that case this throws.

Reply:
done.